index.html

```diff
@@ -35,10 +36,18 @@ <h4>POSTED ON 15 DEC 2013 / DESSERTS</h4>
       <figure id="main-image">
         <img src="img/choco-pizza.png" alt="Baked chocolate pizza" />
       </figure>
-      <p>For the fig-swirl: Melt butter over medium heat in a saucepan. Add Brown sugar and stir to dissolve. Halve all of the figsand toss in the saucepan with water and lemon juice. Cook over medium heat, stirrinf frequently, until you have a chunky-jammymixture. Add salt with one or two stirs, set aside and let cool completely.</p>
-      <p>Ice cream: In a small pot over medium heat, combiine mik, and granulated sugar until sugar is completely dissolved and themilk is just barely lukewarm. Whisk in egg yolks. Set mixture in the fridge and wait until the fig mixture is cooled.</p>
-      <p>Using an ice cream machine, pour liquids into the frozen basin and process according to manufacturer instructions, i.e., let spin and thicken for 20 minutes before adding mascarpone, fig jam mixture,a nd the nuts. Continue the process for +/- 10 minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving.</p>
-      <div class="container"></div>
+        <p>
```

Comment:
This change in spacing shows me that you weren't pulling from master into your feature branch. Make sure that when you merge a pull request into master, you pull those changes onto your branch before you try to merge that branch.